velox/functions/prestosql/aggregates/MinMaxAggregates.cpp

```diff
@@ -177,9 +187,12 @@ class MaxAggregate : public MinMaxAggregate<T> {
   }
 
  private:
-  static constexpr T kInitialValue_{MinMaxTrait<T>::min()};
```

Comment:
Moved from the description
The `static constexpr kInitialValue_` has to non-const because the UnscaledXXX classes do not
support constexpr constructors. The current constructors have a debug check to ensure the values are in range.
This forces us to keep it non-const.

Reply:
What is the error that is thrown if we keep the `constexpr` here?

Reply:
`Constexpr variable 'kInitialValue_' must be initialized by a constant expression`

Reply:
Can you try `static const`? It should be equivalent to `static constexpr` here.